alembic/versions/a3d71270256c_adds_panel_application_fields.py

```diff
@@ -0,0 +1,79 @@
+"""Adds panel application fields
```

Comment:
The names are totally arbitrary, but you're right – we should probably try to make them more meaningful. I'll go back and be more descriptive!